pkg/parser/Makefile

```diff
@@ -1,15 +1,18 @@
-.PHONY: all parser clean
-
-all: fmt parser generate
+.PHONY: all
+all: fmt parser
 
+.PHONY: test
 test: fmt parser
 	sh test.sh
 
-parser: parser.go hintparser.go
+.PHONY: parser
+parser: parser.go hintparser.go generate
 
+.PHONY: genkeyword
```

Comment:
```suggestion
```

I don't think we should mark this phony as the `genkeyword` target actually creates the `genkeyword` file?

Reply:
Thank you. Nice catch.